src/main/kotlin/org/rust/ide/hints/RsChainMethodTypeHintsProvider.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.hints
```

Comment:
```suggestion
@file:Suppress("UnstableApiUsage")

package org.rust.ide.hints
```

Warnings about unstable API are quite annoying